DataFormats/PatCandidates/src/Electron.cc

```diff
@@ -31,6 +32,7 @@ Electron::Electron() :
     ecalRegressionSmear_(-99999.),
     ecalTrackRegressionScale_(-99999.),
     ecalTrackRegressionSmear_(-99999.),
+    miniPFIsolation_(pat::MiniIsolation({9999.,9999.,9999.,9999.})),
```

Comment:
it would be more clear to have these 9999 set in the default constructor of MiniIsolation.